benchmark/tests/obstore/cog_header.py

```diff
@@ -18,15 +18,12 @@ async def fut(store: obs.store.S3Store):
 
     Semaphore allows this function to be called 500 times concurrently
     """
-    r = await obs.get_range_async(store, key, offset=0, length=16384)
-    r.as_bytes()
```

Comment:
Is it important to include the copy from Rust to Python as part of the benchmark? The response implements the buffer protocol, so you can pass it to e.g. `memoryview`

Reply:
yes I'm currently considering a request is "over" when the bytes from the request are pulled into a python variable.  This is to prevent libraries that do this lazily from appearing faster than libraries that don't.  I may reconsider this in the future though.